### ISSUE TITLE
GraphQL connector

### ISSUE THREAD

**Issue by chaffeqa** [Canner/wren-engine#561]
I apologize, I haven't looked closely at how tied to SQL wrenai is, but when I read about it I immediately thought a GraphQL data source may be a good target:
- already has connections defined
- simpler than SQL query language
- similar "column" primitives 

Thought I'd at least ask

**Comment by chaffeqa** [comment]
Yeah after diving in, I realized that WrenAI is very much tied to SQL conversion.  I was hoping maybe there was some separation between the `sql -> data source` that would allow other sources other than SQL.  GraphQL is basically a standardized "graph" of data, which already does the mapping from the "data engine" (SQL, mongo, external API, ect) into a graph that is query-able via JSON API.

To be clear: I don't think that is something Wren should try to support.  Maybe down the road some of the logic could be extracted to another "engine" that could do that, but I'd bet that would be a bunch of work that isn't really in scope.

Thanks for the info!

**Comment by goldmedal** [comment]
Thanks @chaffeqa
